lib/ios.js

```diff
@@ -132,8 +132,19 @@ checks.push(new AuthorizationDbCheck());
 class CarthageCheck extends DoctorCheck {
   async diagnose () {
     let carthagePath = await CarthageDetector.detect();
+
+    let version;
+    if (carthagePath) {
+      try {
+        const {stdout} = await exec(carthagePath, ['version']);
+        version = stdout.trim().split('\n').reverse()[0];
+      } catch (err) {
+        log.warn(err);
+      }
+    }
+
     return carthagePath
-      ? ok(`Carthage was found at: ${carthagePath}`)
+      ? ok(`Carthage was found at: ${carthagePath}${ version ? `. Installed version is: ${version}` : ''}`)
```

Comment:
should we warn if cartage version is too low?

Reply:
We do not need it, so far, I think.

We can show users which version is the latest one, but I'm not sure what version is minimal... I was able to build WDA in current master with carthage 0.28.0 on my env and could run a couple of test cases.
Thus, adding a message like `Carthage was found at: xxxxx. Installed version: xxxx. The latest version: xxxx.` in `ok` method is pretty enough, if we do.